compiler/testData/diagnostics/foreignAnnotationsTests/java8Tests/misc/annotatedPrimitiveArray.kt

```diff
@@ -0,0 +1,11 @@
+// FIR_IDENTICAL
+// SKIP_TXT
+// FILE: J.java
+import org.jetbrains.annotations.Nullable;
+
+public interface J {
+    int @Nullable [] foo();
```

Comment:
BTW are annotations applicable still properly with `int [] @Nullable []` or `int @Nullable [] []`?

Reply:
Yes, you can see that in some of the changed loadJava8 tests.